en: update toc and reorganize some v1.1 content (#59)
pingcap/docs-tidb-operator: en/monitor-a-tidb-cluster.md

## Monitor the TiDB cluster

> You can monitor the TiDB cluster with Prometheus and Grafana. If you create a new TiDB cluster using TiDB Operator, refer to [Monitor a TiDB Cluster using TidbMonitor](monitor-using-tidbmonitor.md). A separate monitoring system is created and configured for each TiDB cluster created by TiDB Operator. The monitoring system runs in the same namespace as the TiDB cluster, and includes two components: Prometheus and Grafana.

Reviewer: ```suggestion
You can monitor the TiDB cluster with Prometheus and Grafana. If you create a new TiDB cluster using TiDB Operator, refer to [Monitor a TiDB Cluster Using TidbMonitor](monitor-using-tidbmonitor.md). A separate monitoring system is created and configured for each TiDB cluster created by TiDB Operator. The monitoring system runs in the same namespace as the TiDB cluster, and includes two components: Prometheus and Grafana.
```

> You can monitor the TiDB cluster with Prometheus and Grafana. If you create a new TiDB cluster using TiDB Operator, refer to [Monitor a TiDB Cluster using TidbMonitor](monitor-using-tidbmonitor.md). A separate monitoring system is created and configured for each TiDB cluster created by TiDB Operator. The monitoring system runs in the same namespace as the TiDB cluster, and includes two components: Prometheus and Grafana.

Reviewer 2: ```suggestion
You can monitor the TiDB cluster with Prometheus and Grafana. When you create a new TiDB cluster using TiDB Operator, you can refer to [Monitor a TiDB Cluster using TidbMonitor](monitor-using-tidbmonitor.md) to deploy a separate monitoring system for the TiDB cluster. The monitoring system has to run in the same namespace as the TiDB cluster. It includes two components: Prometheus and Grafana.
```
